apache/rocketmq/v2/service.proto

```diff
@@ -96,6 +96,7 @@ message ReceiveMessageRequest {
   optional google.protobuf.Duration invisible_duration = 5;
   // For message auto renew and clean
   bool auto_renew = 6;
+  optional google.protobuf.Duration polling_time = 7;
```

Comment:
Why `optional`?

Reply:
It's for version compatibility.

Reply:
How about keeping the same name in `Subscription#long_polling_timeout`.